core/src/main/java/org/testcontainers/images/RemoteDockerImage.java

```diff
@@ -14,13 +15,18 @@
 import org.testcontainers.utility.DockerLoggerFactory;
 import org.testcontainers.utility.LazyFuture;
 
+import java.time.Duration;
+import java.time.Instant;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static java.time.Duration.between;
```

Comment:
Would you mind not importing these methods? 😊

Reply:
Do you mean not using static imports for the methods? Or using some other time API 😂 ?

Reply:
:D not using static imports for non-test stuff :)

Reply:
Fixed!